src/components/artwork/index.tsx

```diff
@@ -15,7 +25,9 @@ export class Artwork extends React.Component<Props, null> {
   render() {
     return (
       <div style={this.props.style}>
-        <Image src={this.props.artwork.image.url} />
+        <Placeholder style={{ paddingBottom: this.props.artwork.image.placeholder }}>
```

Comment:
So `placeholder` is just the height of the image? And is that an absolute dimension?